ui/app/AppLayouts/Wallet/stores/SwapStore.qml

```diff
@@ -8,4 +11,29 @@ QtObject {
     readonly property var accounts: walletSectionAccounts.accounts
     readonly property var flatNetworks: networksModule.flatNetworks
     readonly property bool areTestNetworksEnabled: networksModule.areTestNetworksEnabled
+
```

Comment:
The TODO should be that after the rework on router and send the TransactionStore should include this calls and should be called directly from SwapModalAdaptor instead